cmd/kube-controller-manager/app/core.go

```diff
@@ -267,6 +267,7 @@ func startCloudNodeLifecycleController(ctx context.Context, controllerContext Co
 		controllerContext.ClientBuilder.ClientOrDie("node-controller"),
 		controllerContext.Cloud,
 		controllerContext.ComponentConfig.KubeCloudShared.NodeMonitorPeriod.Duration,
+		controllerContext.ComponentConfig.KubeCloudShared.InstanceExistsGracePeriod.Duration,
```

Comment:
hmm, is this really the place, I think these are legacy and the one you are looking for is in `staging/src/k8s.io/cloud-provider/app/core.go`

Reply:
Yep I also thought this was weird, but KCM is still using the same constructor that k8s.io/cloud-provider does. I had to update its usage in both places